js/app.js

```diff
@@ -1,40 +1,314 @@
+    super(props);
+    this.state = {};
+  }
   render() {
     return (
-      <div>
-        {" "}
-        <i class="fas fa-h1    "></i>{" "}
+      <div className="tab-pane" id="Projects">
+        <h2>Projects</h2>
+        <a
+          href="https://github.com/Hantasmagoria/Hantasmagoria.github.io/blob/master/documentations/overdoze.md"
+          title="Project 1 for Software Engineering Immersive @ General Assembly.
+                              Platform game developed using HTML5 and Javascript.
+                              Assets used are from opengameart.org"
+        >
+          Overdoze
+        </a>
+        <a
+          href="https://overdozews.herokuapp.com/"
+          title="Project 2 for Software Engineering Immersive @ General Assembly.
+                              Database-enabled website for leaderboard and score tracking.
+                              Inspired by design at osu.ppy.sh"
+        >
+          ODDB
+        </a>
+        <a
+          href="https://github.com/Hantasmagoria/UnrehearsedAntihistoricalValedictorian"
+          title="Discord bot catered for the needs of the Invictus discord server. 
+                              Uses the discord.js library, hosted on heroku."
+        >
+          Shana the Discord Bot
+        </a>
+
+        {/* <a href="#">Project 4</a>
+                <a href="#">Project 5</a>
+                <a href="#">Project 6</a>
+                <a href="#">Project 7</a>
+                <a href="#">Project 8</a>
+                <a href="#">Project 9</a>
+                <a href="#">Project 10</a>
+                <a href="#">Project 11</a>  */}
+
+        <a
+          href="https://github.com/Hantasmagoria/mcojn"
+          title="A Javascript Bookmarklet for restructuring the data in the DOM within the facility booking system in MCOnline.sg"
+        >
+          MCOnline JN script
+        </a>
       </div>
     );
   }
 }
+
 class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
   render() {
     return (
-      <div>
-        {" "}
-        <i class="fas fa-h1    "></i>{" "}
+      <div className="tab-pane" id="Contact">
+        <h2>Contact</h2>
+        <p>
+          <a href="mailto:farhan.daud.hyperfusion@gmail.com">
+            farhan.daud.hyperfusion@gmail.com
+          </a>
+        </p>
       </div>
     );
   }
 }
+
+// ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄
+// ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌
+// ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀█░▌
+// ▐░▌       ▐░▌▐░▌       ▐░▌▐░▌       ▐░▌
+// ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄█░▌
+// ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌
+// ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀▀▀ ▐░█▀▀▀▀▀▀▀▀▀
+// ▐░▌       ▐░▌▐░▌          ▐░▌
+// ▐░▌       ▐░▌▐░▌          ▐░▌
+// ▐░▌       ▐░▌▐░▌          ▐░▌
+//  ▀         ▀  ▀            ▀
+
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      mainEntry: 1,
+      currentUser: ""
```

Comment:
leaving this here for future-proofing: will be making a backdoor later on. I've gotta update my content somehow, right?